### ISSUE TITLE
Learning tau on validation set ?

### ISSUE THREAD

**Issue by jiequancui** [facebookresearch/classifier-balancing#5]
Hi, I see that you have tried to learn tau parameter on training set or apply grid search for tau on validation set and training set in the paper. So have you tried learn tau parameter and LWS parameter on validation set or on train+val set ? and Further there seems only test set for iNaturalist 2018, So the val set is splited from training set by yourself ?

**Comment by bingykang** [comment]
We did not learn tau or LWS on validation set. You can try yourself, but I do not think there will be a big difference. 

For iNaturalist, we directly perform tau-normalization on the testing set. When it comes to LWS, we learn the parameter from the training set with class-balanced sampling. For more study on tau, please refer to appendix B.5 in the paper.

**Comment by jiequancui** [comment]
Big Thanks for your reply!
